docs/news.rst

```diff
@@ -911,6 +911,20 @@ Subparsers can be listed with ``--list-subparser``:
 Changes to the tags file format
 ---------------------------------------------------------------------
 
+.. NOT REVIEWED YET
+
+version 3 format for handling meta characters in tag entry
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To make allow to record a name including a tab character to a tag file,
+the tags file format is extended, and we call the new file format version 3.
```

Comment:
*“To allow recording names including tab characters and other control characters in a tag file, version 3 of the file format extends version 2 with extra escape sequences.”*